pkg/glance/const.go

```diff
@@ -90,6 +92,9 @@ const (
 	CachePruner CronJobType = "pruner"
 	//ImageCacheDir -
 	ImageCacheDir = "/var/lib/glance/image-cache"
+
+	// GlanceDBSyncCommand -
+	GlanceDBSyncCommand = "/usr/local/bin/kolla_start"
```

Comment:
we don't need `kolla_set_config` because we already mount `00-config.conf` in the target directory (`/etc/glance/glance.conf.d/`).